src/init.gradle

```diff
@@ -552,6 +629,18 @@ abstract class DependencyTree extends DefaultTask{
         }
     }
 
+    void getExcludedDeps(){
```

Comment:
:x: This isn't a getter, so it shouldn't start with `get`. Suggestion: `computeExcludedDeps()`.